oresat_dxwifi/resources/temperature.py

```diff
@@ -0,0 +1,114 @@
+# Purpose:
+#       Issue: https://github.com/oresat/oresat-dxwifi-software/issues/1.
+#       The temperature "read" by an NTC thermistor is calculated based on
+#           the voltage recieved at the ADC input pin.
+#       The calculations are based on:
+#           https://docs.google.com/document/d/16b_Qd6Pj2IAk5HQQl7JAP1T9Qd6QbKICMd-0swd9L6Y/edit
```

Comment:
I'd rather not link to a private Google doc in an open source project on GitHub. However, I haven't thought of a very straightforward way to include a link. Best idea I've had is to convert the Google doc to a PDF and save it as a gist: https://xai.sh/2019/02/25/Add-arbitrary-files-to-gists.html

I'm not that worried about it. In practice, contributors to this code should have access to the shared drive.

Reply:
Gist was made with the pdf in question and replaced the google doc link.